xml/MAIN-SBP-CSP-UpdateInfra.xml

```diff
@@ -1217,267 +1489,384 @@ systemctl enable rmt-server-regsharing.timer
 							&lt;/RequireAll&gt;
 						&lt;/Directory&gt;
 					</screen>
-				</step>
-				<step>
-					<para> Enable and restart the Region Server service. </para>
-					<screen>
+    </step>
+    <step>
+     <para>
+      Enable and restart the Region Server service.
+     </para>
+<screen>
 						systemctl enable apache2.service
 						systemctl restart apache2.service
 					</screen>
-					<para> The Region Server reads the <filename>regionData.cfg</filename> file as
-						configured in the <filename>regionInfo.cfg</filename> file at start-up. It
-						creates a hash table from the information provided in the
-							<filename>regionData.cfg</filename> file. Depending on the number of IP
-						address ranges, this may result in a relatively large requirement of memory.
-						For estimation purposes, one can use a requirement of 20 MB per octet (254
-						IP addresses). The Region Server also creates a secondary hash table that
-						relates the region names to the RMT server information. The secondary hash
-						table is consulted if the Region Server receives a region hint from the
-						client. </para>
-				</step>
-			</procedure>
-			<para> With the configuration in place the Region Server setup is complete. </para>
-		</sect2>
+     <para>
+      The Region Server reads the <filename>regionData.cfg</filename> file as
+      configured in the <filename>regionInfo.cfg</filename> file at start-up.
+      It creates a hash table from the information provided in the
+      <filename>regionData.cfg</filename> file. Depending on the number of IP
+      address ranges, this may result in a relatively large requirement of
+      memory. For estimation purposes, one can use a requirement of 20 MB per
+      octet (254 IP addresses). The Region Server also creates a secondary hash
+      table that relates the region names to the RMT server information. The
+      secondary hash table is consulted if the Region Server receives a region
+      hint from the client.
```

Comment:
The implementation of the region server has changed. We should probably leave out the details. Suggest:

```The Region Server reads the <filename>regionData.cfg</filename> file as
configured in the <filename>regionInfo.cfg</filename> file at start-up. When a client requests information the provided region hint from the client data is used as the section name in <filename>regionData.cfg</filename> and teh update server data is returned. If no region hint is provided a longest prefix match is used to attempt a client IP look up to match the update server data to the client.```